aptos-move/framework/aptos-stdlib/sources/string_utils.move

```diff
@@ -107,7 +107,7 @@ module aptos_std::string_utils {
         native_format_list(&b"a = {} b = {} c = {}", &l);
     }
 
-    #[testonly]
+    /// #[test_only]
```

Comment:
? why the comment out?

Reply:
We can't actually use #[test_only] on this struct because it will make the module incompatible for upgrades (according to what @wrwg tells me, anyway).  So instead I put it in a comment that will pass through to docs.  Leaving the typo '#[testonly]' attribute could be safe, but might encourage users to keep misspelling, so it's deleted.